src/CattyTests/PlayerEngine/Sensors/Date/TimeMinuteSensorTest.swift

```diff
@@ -65,8 +65,8 @@ final class TimeMinuteSensorTest: XCTestCase {
     }
 
     func testConvertToStandardized() {
-        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1))
-        XCTAssertEqual(100, sensor.convertToStandardized(rawValue: 100))
+        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1, landscapeMode: false))
+        XCTAssertEqual(100, sensor.convertToStandardized(rawValue: 100, landscapeMode: false))
```

Comment:
Please also add a test for `convertToStandardized` when called with `landscapeMode` `true`.